Jenkinsfile.buildMetadata

```diff
@@ -97,12 +114,24 @@ pipeline {
                 environment name: 'APPROVE_DEPLOY', value: 'yes'
             }
             steps {
-                sh "fastlane android promote_Catroid"
+                script {
+                    if (env.flavor == 'Playground') {
+                        echo 'Playground cannot be promoted to production!'
+                    } else {
+                        // deactivated for testing to not publish by accident
+                        //sh 'fastlane android promote_${flavor}'
```

Comment:
still deactivated for commit on purpose? as it was active before

Reply:
Yes this is still deactivated on purpose. This is probably not the cleanest way, but I still kept it in as a comment to keep it documented that this option is still planned in the original workflow and that is is supposed to be right at this step of the process.